spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/SparkCassandraITSpecBase.scala

```diff
@@ -0,0 +1,22 @@
+package com.datastax.spark.connector
+
+import com.datastax.spark.connector.embedded.{SecondEmbeddedCassandra, SparkTemplate}
+import com.datastax.spark.connector.testkit.SharedEmbeddedCassandra
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.cassandra.CassandraSQLContext
+import org.scalatest.{Matchers, FlatSpec, ConfigMap, BeforeAndAfterAll}
+
+
+trait SparkCassandraITSpecBase extends FlatSpec with Matchers with SharedEmbeddedCassandra with SparkTemplate with BeforeAndAfterAll {
```

Comment:
The goal eventually and soon-ish is to have our ITs run in travis CI as well, so just be sure these changes are bomber.